app/views/content/life-as-a-teacher/teaching-as-a-career/change-careers/_article.html.erb

```diff
@@ -0,0 +1,111 @@
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I would advise anyone thinking about changing careers that you will be embraced and more importantly an asset to the profession and students.",
+    name: "Ian, career changer",
+    large: false
+  ) %>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Fund your training</h2>
+    <p>Tax-free bursaries and scholarships of upto <%= v :scholarships_generic_maxshortened %> are available for eligible trainee teachers. You can get these alongside a tuition fee loan and maintenance loan.</p>
+    <p>You may also be able to get extra funding and support if you're:</p>
+    <ul>
+      <li>a parent or carer</li>
+      <li>disabled</li>
+      <li>a veteran</li>
+    </ul>
+    <p>Find out how you can <a href="/funding-and-support">fund your teacher training</a>.</p>
+  </section>
+</div>
+<div class="row">
+  <div class="col-space-l col-space-l-top">
+    <%= render Content::QuoteWithImageComponent.new(
+      title: "Teacher stories - Claire",
+      text: "<p>As a parent and professional with a regular income, the financial support available to me as a trainee physics teacher made the option of re-training seem not only viable, but attractive.</p><a href='/life-as-a-teacher/teaching-as-a-career/becoming-a-parent-sparked-my-interest-in-teaching'>Read more about Claire's career change to teaching</a>",
+      quotes: true,
+      background_color: "blue",
+      heading_color: "pink",
+    ) %>
+  </div>
+</div>
+<div class="row inset">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Support for career changers</h2>
+    <h3>Finding and applying for training</h3>
+    <p>Our experienced teaching advisers can support you through the switch if you're thinking about changing careers. You can chat to an adviser by phone, text or email.</p>
+    <p>Find out more about <a href="/teacher-training-adviser/sign_up/identity">getting a teacher training adviser</a>.</p>
+    <p><a href="https://nowteach.org.uk/">Now Teach</a> is a service that specifically helps people get into teaching who are changing career.</p>
+    <h3>During your training and early career</h3>
+    <p>Throughout teacher training, you'll have the support of a mentor.</p>
+    <p>Your mentor will:</p>
+    <ul>
+      <li>guide you through your training, sharing learning and career development opportunities with you</li>
+      <li>meet with you regularly to make sure your training is a positive experience</li>
+    </ul>
+    <p>You'll also be part of a welcoming school community where you'll learn from fellow teachers through taught sessions, seminars and workshops.</p>
+    <p>Once you've completed your teacher training, you'll continue to be supported during your first 2 years of teaching. Find out what support you'll get as <a href="/life-as-a-teacher/teaching-as-a-career/early-career-teachers">an early career teacher</a>.</p>
+  </section>
+</div>
```

Comment:
Worth adding the adviser promo at the end? Might be able to use the partial one set up for subject pages